Quaver.Shared/QuaverGame.cs

```diff
@@ -626,6 +627,21 @@ private void HandleKeyPressCtrlS()
             }
         }
 
+        /// <summary>
+        ///    Handles when the user holds either Alt (ALT) button and presses Enter
+        /// </summary>
+        private void HandleKeyPressAltEnter()
+        {
+            // Check for modifier keys
+            if (!(KeyboardManager.CurrentState.IsKeyDown(Keys.LeftAlt) || KeyboardManager.CurrentState.IsKeyDown(Keys.RightAlt)))
+                return;
+
+            if (!KeyboardManager.IsUniqueKeyPress(Keys.Enter))
+                return;
+
+            Graphics.ToggleFullScreen();
```

Comment:
Use `ConfigManager.WindowFullScreen.Value = !ConfigManager.WindowFullScreen.Value;` instead, as this will update the config and automatically toggle fullscreen.